BrowserKit/Sources/ComponentLibrary/Buttons/SecondaryRoundedButton.swift

```diff
@@ -48,10 +48,10 @@ public class SecondaryRoundedButton: ResizableButton, ThemeApplicable {
         }
         updatedConfiguration.baseForegroundColor = foregroundColor
 
-        updatedConfiguration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer { incoming in
+        updatedConfiguration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer { [weak foregroundColor] incoming in
```

Comment:
Thanks @amanjeetsingh150 🙏 Great work. Unfortunately there is a Swiftlint warning for this line which fails the build (`Line should be 125 characters or less; currently it has 139 characters`). I suggest you do the following to fix it: 
```
let transformer = UIConfigurationTextAttributesTransformer { [weak foregroundColor] incoming in
	var container = incoming

	container.foregroundColor = foregroundColor
	container.font = DefaultDynamicFontHelper.preferredBoldFont(
		withTextStyle: .callout,
		size: UX.buttonFontSize
	)
	return container
}
updatedConfiguration.titleTextAttributesTransformer = transformer
```

Reply:
Hey @thatswinnie Thanks for the suggestion. Fixed 🙌 

Also verified, the report again 💯